library/Rules/Email.php

```diff
@@ -5,6 +5,14 @@ class Email extends AbstractRule
 {
     public function validate($input)
     {
-        return is_string($input) && filter_var($input, FILTER_VALIDATE_EMAIL);
+        if (class_exists('\Egulias\EmailValidator\EmailValidator')) {
+            $strictValidator = new \Egulias\EmailValidator\EmailValidator();
```

Comment:
PS: When testing use a mock of `\Egulias\EmailValidator\EmailValidator` and pass it as dependency to the rule instead of use the original class, them you can assert if Validation is making the correct call to EmailValidator.